lib/workflow/client.js

```diff
@@ -129,6 +129,14 @@ class Workflow {
     };
   }
 
+  profileTasks(profileId, establishmentId) {
+    let url = `/profile-tasks/${profileId}`;
+    if (establishmentId) {
+      url = `${url}/${establishmentId}`;
```

Comment:
Minor niggle, but this feels like it would be better suited to a query string parameter than a URL fragment.

Reply:
I think you're right